.circleci/config.yml

```diff
@@ -72,42 +61,31 @@ run_linter: &run_linter
       pylint spython
 
 test_spython: &test_spython
-  name: Test Singularity Python (Singularity Version 2 and 3)
-  command: |
-      cd ~/repo/spython
-      pytest -k 'not test_oci'
-
-test_spython_3: &test_spython_3
-  name: Test Singularity Python (Singularity Version 3 Only)
-  command: |
-      cd ~/repo/spython
-      pytest -k 'test_oci'
-
+  name: Test Singularity Python
+  command: pytest ~/repo/spython
 
 jobs:
-  test-singularity-3-python-3:
+  test-singularity-3-python-3: &latest_singularity
     machine: true
     working_directory: ~/repo
     steps:
       - checkout
       - restore_cache:
           keys:
-          - v1-dependencies-py3
-      - run: *install_python_3
+          - v2-dependencies
```

Comment:
If there is already a cache with this name, it might run into issue (I believe this happened before).

Reply:
Which issue? I install both Python versions into `~/conda` (which is cached) but in subfolders. Nothing bad will happen this way

Reply:
I remember when we first started working, we had a cache I think for Python 3 that was already used for Python 2, so the tests appeared to be working but they were all Python 2. This might not be the case, I just want to double check (and I'll do this when it runs). :)